modules/list_books.rb

```diff
@@ -0,0 +1,6 @@
+module ListBooks
```

Comment:
- Fantastic! :+1: This method ONLY lists the books and does nothing else such as requesting inputs or validation. This is exactly how each method/class should be structured to meet up with the SRP